frontend/appflowy_flutter/lib/plugins/database/widgets/cell/editable_cell_skeleton/url.dart

```diff
@@ -135,6 +144,83 @@ class _GridURLCellState extends GridEditableTextCell<EditableURLCell> {
   String? onCopy() => cellBloc.state.content;
 }
 
+class MobileURLEditor extends StatelessWidget {
+  const MobileURLEditor({
+    super.key,
+    required this.bloc,
+    required this.textEditingController,
+  });
+
+  final URLCellBloc bloc;
+  final TextEditingController textEditingController;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const VSpace(
+          4.0,
+        ),
```

Comment:
```suggestion
        const VSpace(4.0),
```
PS. All my "style" comments in all PRs are always nits :joy: